
        <html> 
            <head>
                <title>Revana Report - GFI1B</title>
                <link
                    href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
                    rel="stylesheet"
                    integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
                    crossorigin="anonymous"
                />
                <script
                    src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"
                    integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4"
                    crossorigin="anonymous"
                ></script>
                <style>
                    .sv-pos-table,
                    .cna-pos-table {
                        table-layout: fixed;
                    }

                    .sv-pos-table td:first-child,
                    .sv-pos-table th:first-child  {
                        width: 15em;
                    }
                </style>
            </head>
            <body>
                <div class="container">
                    <div class = "card my-4 analysis_by_gene_proteine_coding">
                        <h5 class="card-header">GFI1B
                            <button class="btn btn-secondary badge float-end ms-2" data-bs-toggle="collapse" data-bs-target="#card-body-gene-GFI1B" aria-expanded="true" aria-controls="card-body-gene-GFI1B">show/hide</button>
                            <span class="badge bg-danger float-end">By gene</span>
                        </h5>
                        <div class="collapse show" id="card-body-gene-GFI1B" style="">
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="p-4 w-50 border-end">
                                    <p>Chromosome: chr9</p>
                                    <p>Position: 135 820 932 - 135 867 083</p>
                                    <p>Width: 46 152</p>
                                    <p>Strand: +</p>
                                    <p>Gene Type: protein_coding</p>
                                    <p>Imprinting Status: no_imprinting</p>
                                    <p>Cancer Gene: no</p>
                                    <p>Role in Cancer: -</p>
                                    <p>TAD region: 135 280 001 - 135 920 000</p>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/mutation_type_matrix_plot___GFI1B.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/passed_filter_all_samples_upsetr___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/mutation_type_all_samples_upsetr___GFI1B.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/expression_by_sample_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/expression_by_group_plot___GFI1B.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/cov_ratio_by_sample_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/expression_copy_number_normalized_by_group_plot___GFI1B.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/ASE_detection_type_matrix_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/allelic_imbalance_by_group_plot___GFI1B.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/allelic_imbalance_expression_dot_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/allelic_imbalance_expression_dot_plot___GFI1B_SV_CNA_highlighted.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/allelic_imbalance_expression_copy_number_normalized_dot_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50">
                                    <img class ="w-100 p-2" src="figures/allelic_imbalance_expression_copy_number_normalized_dot_plot___GFI1B_SV_CNA_highlighted.svg"/>
                                </div>
                            </div>
                            <div class="d-flex flex-wrap border-bottom">
                                <div class="w-50 border-end">
                                    <img class ="w-100 p-2" src="figures/gene_locus_plot___GFI1B.svg"/>
                                </div>
                                <div class="w-50 p-2">
                                    <div id="igv-window"></div>
                                    <div id="igv-btn-container" class="w-100 border border-primary h-100 d-flex align-items-center justify-content-center">
                                        <button class="btn btn-lg btn-primary" id="igv-start-btn" type="button">START IGV</button>
                                    </div>
                                </div>
                            </div>
                            <div class="p-3 border-bottom">
                                <h3>GeneHancer
                                    <button
                                        class="btn btn-secondary badge float-end ms-2 collapsed"
                                        data-bs-toggle="collapse"
                                        data-bs-target="#gene-card-GFI1B-genehancer-data"
                                        aria-expanded="false"
                                        aria-controls="gene-card-GFI1B-genehancer-data">show/hide</button>
                                </h3>
                                <div class="border-bottom mb-3 collapse" id="gene-card-GFI1B-genehancer-data"><h4>GH09J132885 - (chr9:135759988-135765786)</h4><p>Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_genehancer </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB137 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132890 - (chr9:135766388-135766787)</h4><p>Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132896 - (chr9:135771498-135773092)</h4><p>Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132908 - (chr9:135783788-135784587)</h4><p>Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132944 - (chr9:135818187-135821188)</h4><p>Promoter/Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132977 - (chr9:135852465-135854406)</h4><p>Promoter/Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_genehancer </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB137 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>GH09J132981 - (chr9:135857431-135863092)</h4><p>Enhancer - within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_genehancer </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB45 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                                </div>
                            </div>
                            <div class="p-3 border-bottom">
                                <h3>ChIP-Seq
                                    <button
                                        class="btn btn-secondary badge float-end ms-2 collapsed"
                                        data-bs-toggle="collapse"
                                        data-bs-target="#gene-card-GFI1B-chipseq-data"
                                        aria-expanded="false"
                                        aria-controls="gene-card-GFI1B-chipseq-data">show/hide</button>
                                </h3>
                                <div class="border-bottom mb-3 collapse" id="gene-card-GFI1B-chipseq-data"><h4>chr9:135404429-135410569</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_chipseq </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB108 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135444905-135541563</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_REG </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_chipseq </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB233 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB237 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135672353-135676701</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 5 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135307682-135312154</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135607521-135610146</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>somatic SNVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n_within_chipseq </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135619580-135622285</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__BETWEEN </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:135804714-135808809</h4><p>within gene TAD</p>
                <h5>SVs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_SV </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 3 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH&lt;-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__IN_GENE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH-&gt;ON_SHARED_CHROM__OUTSIDE_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__HIGH_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__OUTSIDE_TAD__LOW_OF_BOTH-&gt;ON_SHARED_CHROM__IN_SHARED_TAD__LOW_OF_BOTH </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
<h4>chr9:131886411-131891784</h4><p>outside gene TAD</p>
                <h5>CNAs</h5><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_ID </th>
   <th style="text-align:left;"> position_category_CNA </th>
   <th style="text-align:left;"> cis_activated_gene </th>
   <th style="text-align:right;"> n </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> ON_SHARED_CHROM__IN_REG_TAD__BETWEEN-&gt;ON_SHARED_CHROM__IN_GENE_TAD__BETWEEN </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:right;"> 1 </td>
  </tr>
</tbody>
</table>
                                </div>
                            </div>
                            <div class="p-3">
                            
                <h3>ICGC_MB247<span class="ms-3 badge bg-success fs-6 fw-light align-top">cis-activated</span><span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB247-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB247-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB247-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB247" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB247">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB247').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB247').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB247"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB247"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB247"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB247"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB247"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB247"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB247 </td>
   <td style="text-align:left;"> group_3 </td>
   <td style="text-align:right;"> 13.3 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> 0.00469385084298183,0.000165866036467018,0.00469385084298183,3.06189454325645e-08 </td>
   <td style="text-align:left;"> 0.5,0.5,0.5,0.5 </td>
   <td style="text-align:left;"> diploid,diploid,diploid,diploid </td>
   <td style="text-align:right;"> 0.0001028 </td>
   <td style="text-align:right;"> 0.5 </td>
   <td style="text-align:right;"> 0.0125967 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>1</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>1</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>2</td>
                                    <td>1</td>
                                    <td> - </td>
                                    <td>2</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135485069 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135872032 </td>
   <td style="text-align:left;"> INV </td>
   <td style="text-align:left;"> INV </td>
  </tr>
</tbody>
</table><h5>Somatic SNVs</h5><table class="table table-striped table-bordered somatic-snv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> snv_chrom </th>
   <th style="text-align:right;"> snv_pos </th>
   <th style="text-align:left;"> ref </th>
   <th style="text-align:left;"> alt </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135591141 </td>
   <td style="text-align:left;"> T </td>
   <td style="text-align:left;"> A </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB9<span class="ms-3 badge bg-success fs-6 fw-light align-top">cis-activated</span><span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB9-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB9-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB9-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB9" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB9">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB9').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB9').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB9"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB9"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB9"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB9"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB9"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB9"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB9 </td>
   <td style="text-align:left;"> group_3 </td>
   <td style="text-align:right;"> 6.46 </td>
   <td style="text-align:right;"> 10 </td>
   <td style="text-align:right;"> 10 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> 0.000165866036466994,0.000920318066595282,0.000373479238582212,0.00117493110878341,6.55285117623248e-05,9.41288189490847e-05,0.000165866036466994,0.000373479238582212,1.00551903474052e-14,2.05820534394824e-10 </td>
   <td style="text-align:left;"> 0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5 </td>
   <td style="text-align:left;"> diploid,diploid,diploid,diploid,diploid,diploid,diploid,diploid,diploid,diploid </td>
   <td style="text-align:right;"> 5.9e-06 </td>
   <td style="text-align:right;"> 0.5 </td>
   <td style="text-align:right;"> 0.0003917 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>6</td>
                                    <td>6</td>
                                    <td>6</td>
                                    <td>6</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>6</td>
                                    <td>6</td>
                                    <td>6</td>
                                    <td>6</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>1</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>1</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>13</td>
                                    <td>12</td>
                                    <td> - </td>
                                    <td>13</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135469280 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135545734 </td>
   <td style="text-align:left;"> INV </td>
   <td style="text-align:left;"> INV </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135469284 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135545734 </td>
   <td style="text-align:left;"> INV </td>
   <td style="text-align:left;"> INV </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135482524 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135876029 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135482763 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135876029 </td>
   <td style="text-align:left;"> TRA </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135577312 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135879940 </td>
   <td style="text-align:left;"> INV </td>
   <td style="text-align:left;"> INV </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135577519 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135879940 </td>
   <td style="text-align:left;"> INV </td>
   <td style="text-align:left;"> INV </td>
  </tr>
</tbody>
</table>
                        <h5>CNAs</h5><table class="table table-striped table-bordered cna-table">
 <thead>
  <tr>
   <th style="text-align:left;"> cna_chrom </th>
   <th style="text-align:right;"> cna_start </th>
   <th style="text-align:right;"> cna_end </th>
   <th style="text-align:right;"> copy_number </th>
   <th style="text-align:left;"> CNA_type </th>
   <th style="text-align:right;"> log2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135469281 </td>
   <td style="text-align:right;"> 135469284 </td>
   <td style="text-align:right;"> 0.000000 </td>
   <td style="text-align:left;"> HomoDel </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135469285 </td>
   <td style="text-align:right;"> 135482524 </td>
   <td style="text-align:right;"> 1.170296 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135482525 </td>
   <td style="text-align:right;"> 135482763 </td>
   <td style="text-align:right;"> 2.850959 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135545735 </td>
   <td style="text-align:right;"> 135577312 </td>
   <td style="text-align:right;"> 0.596952 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135577313 </td>
   <td style="text-align:right;"> 135577519 </td>
   <td style="text-align:right;"> 0.000000 </td>
   <td style="text-align:left;"> HomoDel </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135876030 </td>
   <td style="text-align:right;"> 135879940 </td>
   <td style="text-align:right;"> 2.518763 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:right;"> NA </td>
  </tr>
</tbody>
</table><h5>Somatic SNVs</h5><table class="table table-striped table-bordered somatic-snv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> snv_chrom </th>
   <th style="text-align:right;"> snv_pos </th>
   <th style="text-align:left;"> ref </th>
   <th style="text-align:left;"> alt </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135581358 </td>
   <td style="text-align:left;"> C </td>
   <td style="text-align:left;"> T </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB230<span class="ms-3 badge bg-success fs-6 fw-light align-top">cis-activated</span><span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB230-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB230-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB230-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB230" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB230">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB230').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB230').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB230"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB230"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB230"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB230"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB230"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB230"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB230 </td>
   <td style="text-align:left;"> group_4 </td>
   <td style="text-align:right;"> 11.51 </td>
   <td style="text-align:right;"> 2 </td>
   <td style="text-align:right;"> 2 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> 1.64889410760754e-07,2.05820534394824e-10 </td>
   <td style="text-align:left;"> 0.5,0.5 </td>
   <td style="text-align:left;"> diploid,diploid </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0.5 </td>
   <td style="text-align:right;"> 1.3e-06 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>2</td>
                                    <td>2</td>
                                    <td> - </td>
                                    <td>2</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135489513 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135849530 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
</tbody>
</table>
                        <h5>CNAs</h5><table class="table table-striped table-bordered cna-table">
 <thead>
  <tr>
   <th style="text-align:left;"> cna_chrom </th>
   <th style="text-align:right;"> cna_start </th>
   <th style="text-align:right;"> cna_end </th>
   <th style="text-align:right;"> copy_number </th>
   <th style="text-align:left;"> CNA_type </th>
   <th style="text-align:right;"> log2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135489514 </td>
   <td style="text-align:right;"> 135849530 </td>
   <td style="text-align:right;"> -0.8024467 </td>
   <td style="text-align:left;"> HomoDel </td>
   <td style="text-align:right;"> NA </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB70<span class="ms-3 badge bg-success fs-6 fw-light align-top">cis-activated</span><span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB70-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB70-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB70-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB70" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB70">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB70').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB70').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB70"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB70"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB70"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB70"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB70"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB70"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB70 </td>
   <td style="text-align:left;"> group_4 </td>
   <td style="text-align:right;"> 21.13 </td>
   <td style="text-align:right;"> 2 </td>
   <td style="text-align:right;"> 2 </td>
   <td style="text-align:right;"> 2 </td>
   <td style="text-align:left;"> 0,0 </td>
   <td style="text-align:left;"> 0.618020541549953,0.520744680851064 </td>
   <td style="text-align:left;"> cnv,cnv </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0.5693826 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> TRUE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>3</td>
                                    <td>3</td>
                                    <td>3</td>
                                    <td>3</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>4</td>
                                    <td>4</td>
                                    <td> - </td>
                                    <td>4</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135470241 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135887872 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
</tbody>
</table>
                        <h5>CNAs</h5><table class="table table-striped table-bordered cna-table">
 <thead>
  <tr>
   <th style="text-align:left;"> cna_chrom </th>
   <th style="text-align:right;"> cna_start </th>
   <th style="text-align:right;"> cna_end </th>
   <th style="text-align:right;"> copy_number </th>
   <th style="text-align:left;"> CNA_type </th>
   <th style="text-align:right;"> log2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135470242 </td>
   <td style="text-align:right;"> 135887872 </td>
   <td style="text-align:right;"> 6.357921 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135887873 </td>
   <td style="text-align:right;"> 140779965 </td>
   <td style="text-align:right;"> 5.273185 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 132509971 </td>
   <td style="text-align:right;"> 135470241 </td>
   <td style="text-align:right;"> 5.195489 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB110<span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB110-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB110-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB110-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB110" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB110">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB110').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB110').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB110"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB110"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB110"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB110"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB110"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB110"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB110 </td>
   <td style="text-align:left;"> group_4 </td>
   <td style="text-align:right;"> 0.01 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0.9256152 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> FALSE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>3</td>
                                    <td>3</td>
                                    <td>3</td>
                                    <td>3</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>4</td>
                                    <td>4</td>
                                    <td>4</td>
                                    <td>4</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>2</td>
                                    <td>0</td>
                                    <td>2</td>
                                    <td>2</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>9</td>
                                    <td>7</td>
                                    <td> - </td>
                                    <td>9</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135440336 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135455421 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135440341 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135455421 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135454120 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135712136 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
</tbody>
</table>
                        <h5>CNAs</h5><table class="table table-striped table-bordered cna-table">
 <thead>
  <tr>
   <th style="text-align:left;"> cna_chrom </th>
   <th style="text-align:right;"> cna_start </th>
   <th style="text-align:right;"> cna_end </th>
   <th style="text-align:right;"> copy_number </th>
   <th style="text-align:left;"> CNA_type </th>
   <th style="text-align:right;"> log2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135440337 </td>
   <td style="text-align:right;"> 135440341 </td>
   <td style="text-align:right;"> 0.000000 </td>
   <td style="text-align:left;"> HomoDel </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135440342 </td>
   <td style="text-align:right;"> 135454120 </td>
   <td style="text-align:right;"> 4.740415 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135454121 </td>
   <td style="text-align:right;"> 135455421 </td>
   <td style="text-align:right;"> 4.977455 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135455422 </td>
   <td style="text-align:right;"> 135712136 </td>
   <td style="text-align:right;"> 5.062247 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
</tbody>
</table><h5>Somatic SNVs</h5><table class="table table-striped table-bordered somatic-snv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> snv_chrom </th>
   <th style="text-align:right;"> snv_pos </th>
   <th style="text-align:left;"> ref </th>
   <th style="text-align:left;"> alt </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135454517 </td>
   <td style="text-align:left;"> C </td>
   <td style="text-align:left;"> T </td>
  </tr>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135608341 </td>
   <td style="text-align:left;"> C </td>
   <td style="text-align:left;"> G </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB165<span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB165-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB165-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB165-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB165" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB165">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB165').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB165').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB165"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB165"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB165"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB165"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB165"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB165"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB165 </td>
   <td style="text-align:left;"> group_4 </td>
   <td style="text-align:right;"> 0.04 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0.2703321 </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> FALSE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td> - </td>
                                    <td>1</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>CNAs</h5><table class="table table-striped table-bordered cna-table">
 <thead>
  <tr>
   <th style="text-align:left;"> cna_chrom </th>
   <th style="text-align:right;"> cna_start </th>
   <th style="text-align:right;"> cna_end </th>
   <th style="text-align:right;"> copy_number </th>
   <th style="text-align:left;"> CNA_type </th>
   <th style="text-align:right;"> log2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135340069 </td>
   <td style="text-align:right;"> 135359998 </td>
   <td style="text-align:right;"> 4.990848 </td>
   <td style="text-align:left;"> DUP </td>
   <td style="text-align:right;"> NA </td>
  </tr>
</tbody>
</table></div>

                <h3>ICGC_MB175<span class="ms-3 badge bg-primary fs-6 fw-light align-top">SVs/CNAs</span>
                <button
                    class="btn btn-secondary badge float-end ms-2 collapsed"
                    data-bs-toggle="collapse"
                    data-bs-target="#gene-card-GFI1B-sample-ICGC_MB175-data"
                    aria-expanded="true"
                    aria-controls="gene-card-GFI1B-sample-ICGC_MB175-data">show/hide</button></h3>
                <div class="border-bottom mb-3 collapse show" id="gene-card-GFI1B-sample-ICGC_MB175-data">
                    <div class="igc-btns-container">
                        <h5>IGV</h5>
                        <div id="load-track-btn-container-ICGC_MB175" class="pb-3" style="display: none">
                            <p class="m-0">&#10003; Bam Files loaded</p>
                            <button class="btn btn-sm btn-primary load-track-btn" data-sample-id="ICGC_MB175">
                                Load Sample in IGV
                            </button>
                        </div>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-bam-ICGC_MB175').click();">
                            Open Bam File
                        </button>
                        <button class="btn btn-sm btn-secondary" onclick="document.getElementById('file-input-index-ICGC_MB175').click();">
                            Open Index File
                        </button>
                        <input
                            id="file-input-bam-ICGC_MB175"
                            class="file-input-bam"
                            type="file"
                            data-sample-id="ICGC_MB175"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB175"
                            style="display: none"
                        />
                        <input
                            id="file-input-index-ICGC_MB175"
                            class="file-input-index"
                            type="file"
                            data-sample-id="ICGC_MB175"
                            data-load-track-btn-container-id="load-track-btn-container-ICGC_MB175"
                            style="display: none"
                        />
                    </div>
                        <h5>General</h5>
                        <div class="overflow-scroll"><table class="table table-striped table-bordered">
 <thead>
  <tr>
   <th style="text-align:left;"> sample_id </th>
   <th style="text-align:left;"> subgroup </th>
   <th style="text-align:right;"> FPKM </th>
   <th style="text-align:right;"> n_markers </th>
   <th style="text-align:right;"> n_ASE_markers </th>
   <th style="text-align:right;"> n_cnv_markers </th>
   <th style="text-align:left;"> p_all </th>
   <th style="text-align:left;"> delta_abs_all </th>
   <th style="text-align:left;"> copynumber_tag_all </th>
   <th style="text-align:right;"> combined_p_value </th>
   <th style="text-align:right;"> mean_delta_abs </th>
   <th style="text-align:right;"> ABH </th>
   <th style="text-align:left;"> ASE_marker_run_overlap </th>
   <th style="text-align:left;"> passed_FPKM_filter </th>
   <th style="text-align:left;"> passed_ASE_filter </th>
   <th style="text-align:left;"> ASE_filter_rescued_oncogene </th>
   <th style="text-align:left;"> passed_mean_delta_abs_filter </th>
   <th style="text-align:left;"> OHE_used_reference </th>
   <th style="text-align:right;"> OHE_used_pvalue </th>
   <th style="text-align:left;"> passed_OHE_filter </th>
   <th style="text-align:left;"> cis_activated_gene </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> ICGC_MB175 </td>
   <td style="text-align:left;"> group_4 </td>
   <td style="text-align:right;"> 0.07 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 0 </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:left;">  </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:right;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> FALSE </td>
   <td style="text-align:left;"> NA </td>
   <td style="text-align:left;"> ref_filtered </td>
   <td style="text-align:right;"> 0.0060756 </td>
   <td style="text-align:left;"> TRUE </td>
   <td style="text-align:left;"> FALSE </td>
  </tr>
</tbody>
</table></div>
                        <h5>Mutation Counts</h5>
                        <table class="table table-striped table-bordered">
                            <thead>
                                <tr>
                                    <th>Variant Type</th>
                                    <th>n matched via gene TAD</th>
                                    <th>n matched via GeneHancer</th>
                                    <th>n affecting ChIPSeq</th>
                                    <th>total</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th>SVs</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                    <td>1</td>
                                </tr>
                                <tr>
                                    <th>CNAs</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                    <td>0</td>
                                </tr>
                                <tr>
                                    <th>relevant somatic SNVs/InDels</th>
                                    <td>0</td>
                                    <td>0</td>
                                    <td> - </td>
                                    <td> - </td>
                                </tr>
                                <tr>
                                    <th>all Mutations</th>
                                    <td>1</td>
                                    <td>1</td>
                                    <td> - </td>
                                    <td>1</td>
                                </tr>
                            </tbody>
                        </table>
                        <h5>SVs</h5><table class="table table-striped table-bordered sv-table">
 <thead>
  <tr>
   <th style="text-align:left;"> sv_break1_chrom </th>
   <th style="text-align:right;"> sv_break1_pos </th>
   <th style="text-align:left;"> sv_break2_chrom </th>
   <th style="text-align:right;"> sv_break2_pos </th>
   <th style="text-align:left;"> sv_type </th>
   <th style="text-align:left;"> eventInversion </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135894809 </td>
   <td style="text-align:left;"> chr9 </td>
   <td style="text-align:right;"> 135896575 </td>
   <td style="text-align:left;"> DEL </td>
   <td style="text-align:left;"> NORMAL </td>
  </tr>
</tbody>
</table></div>
                            </div>
                        </div>
                    </div>
                </div>
                <script src="https://cdn.jsdelivr.net/npm/igv@2.10.5/dist/igv.min.js"></script>
                <script>
                    const igvDiv = document.getElementById("igv-window");
                    class IgvController {
                        constructor(igvOutElement) {
                            this.igvOutElement = igvOutElement;
                            this.browser = null;
                            this.isIgvOpen = false;
                            this.sampleBamFiles = [];
                            this.sampleBamIndexFiles = [];
                            this.onOpenIGVCompleteCallback = () => {}
                        }

                        setLocus(locus) {
                            console.log(`Going to ${locus}`)
                            if(this.isIgvOpen){
                                this.browser.search(locus)
                            }else{
                                this.locus = locus
                            }
                        }

                        setLocusBySpotAndPadding(chrom, location, padding) {
                            const start = location - padding;
                            const end = location + padding;
                            const locus = `${chrom}:${start}-${end}`
                            this.setLocus(locus)
                        }

                        setBrowser(browser){
                            this.browser = browser;
                        }

                        setOnOpenIGVCompleteCallback(callback){
                            this.onOpenIGVCompleteCallback = callback;
                        }

                        getOptions() {
                            const options = {
                                genome: "hg19",
                                locus: this.locus,
                            };
                            return options;
                        }

                        openIGV() {
                            const igvOut = this.igvOutElement;
                            const options = this.getOptions();
                            const onBrowserCreated = (browser) => {
                                console.log("Created IGV browser");
                                this.setBrowser(browser);
                                this.isIgvOpen = true;
                                this.onOpenIGVCompleteCallback();
                            };
                            igv.createBrowser(this.igvOutElement, options).then(onBrowserCreated);
                        }

                        loadTrack({label, file, indexFile, callback = () => {}}){
                            this.browser.loadTrack({
                                label: label,
                                url: file,
                                indexURL: indexFile,
                                format: "bam"
                            })
                            .then(callback)
                            .catch((e) => console.error(e))
                        }

                        addSampleBamFile(sampleId, file){
                            const sampleBamFiles = this.sampleBamFiles.filter((obj) => (!(obj.sampleId === sampleId)));
                            sampleBamFiles.push({
                                sampleId,
                                file
                            })
                            this.sampleBamFiles = sampleBamFiles
                        }

                        addSampleBamIndexFile(sampleId, file){
                            const sampleBamIndexFiles = this.sampleBamIndexFiles.filter((obj) => (!(obj.sampleId === sampleId)));
                            sampleBamIndexFiles.push({
                                sampleId,
                                file
                            })
                            this.sampleBamIndexFiles = sampleBamIndexFiles
                        }

                        getSampleBamFileBySampleId(sampleId) {
                            const sampleBamFileObj = this.sampleBamFiles.find(
                                (obj) => obj.sampleId === sampleId
                            );
                            return sampleBamFileObj.file;
                        }

                        getSampleBamIndexFileBySampleId(sampleId) {
                            const sampleBamIndexFileObj = this.sampleBamIndexFiles.find(
                                (obj) => obj.sampleId === sampleId
                            );
                            return sampleBamIndexFileObj.file;
                        }

                        areSampleFilesAvailable(sampleId){
                            const sampleBamFile = this.getSampleBamFileBySampleId(sampleId);
                            const sampleBamIndexFile = this.getSampleBamIndexFileBySampleId(sampleId);
                            return((!!sampleBamFile)&&(!!sampleBamIndexFile))
                        }

                        loadTrackBySampleId(sampleId){
                            if(this.areSampleFilesAvailable(sampleId) && this.isIgvOpen){
                                const bamFile = this.getSampleBamFileBySampleId(sampleId);
                                const bamIndexFile = this.getSampleBamIndexFileBySampleId(sampleId);

                                this.loadTrack({
                                    label: sampleId, 
                                    file: bamFile,
                                    indexFile: bamIndexFile,
                                    callback: () => {console.log("track loaded")}
                                })
                            }else{
                                if(!this.areSampleFilesAvailable(sampleId)){
                                    window.alert("sample files not available")
                                }
                                if(!this.isIgvOpen){
                                    window.alert("Please start IGV Plugin first!")
                                }
                            }
                        }
                    }

                    const igvController = new IgvController(igvDiv);
                    igvController.setLocus("chr9:135280001-135920000")



                    const igvBtn = document.getElementById("igv-start-btn");
                    const igvBtnContainer = document.getElementById("igv-btn-container");
                    const bamFileInputsElements = Array.from(
                        document.getElementsByClassName("file-input-bam")
                    );
                    const bamIndexFileInputsElements = Array.from(
                        document.getElementsByClassName("file-input-index")
                    );
                    const loadTrackBtnElements = Array.from(
                        document.getElementsByClassName("load-track-btn")
                    );
                    const svTableRowElements = Array.from(
                        document.querySelectorAll("table.sv-table > tbody > tr")
                    );
                    const cnaTableRowElements = Array.from(
                        document.querySelectorAll("table.cna-table > tbody > tr")
                    );
                    const somaticSnvTableRowElements = Array.from(
                        document.querySelectorAll("table.somatic-snv-table > tbody > tr")
                    );



                    const hideElement = (element) => {
                        element.style.display = "none";
                    }

                    const displayElement = (element) => {
                        element.style.display = "block";
                    }

                    const removeElement = (element) => {
                        element.remove();
                    }



                    const handleIgvBtnClick = () => {
                        igvController.openIGV();
                        removeElement(igvBtnContainer);
                    }

                    const handleBamFileInputChange = (e) => {
                        const file = e.target.files[0];
                        const sampleId = e.target.dataset.sampleId;
                        igvController.addSampleBamFile(sampleId, file)

                        if(igvController.areSampleFilesAvailable(sampleId)) {
                            const loadTrackBtnContainerElement = document.getElementById(e.target.dataset.loadTrackBtnContainerId)
                            displayElement(loadTrackBtnContainerElement)
                        }
                    }

                    const handleBamIndexFileInputChange = (e) => {
                        const file = e.target.files[0];
                        const sampleId = e.target.dataset.sampleId;
                        igvController.addSampleBamIndexFile(sampleId, file)

                        if(igvController.areSampleFilesAvailable(sampleId)) {
                            const loadTrackBtnContainerElement = document.getElementById(e.target.dataset.loadTrackBtnContainerId)
                            displayElement(loadTrackBtnContainerElement)
                        }
                    }

                    const handleLoadTrackBtnClick = (e) => {
                        const sampleId = e.target.dataset.sampleId;
                        igvController.loadTrackBySampleId(sampleId)
                    }



                    igvBtn.addEventListener("click", handleIgvBtnClick);

                    bamFileInputsElements.forEach((element) => {
                        element.addEventListener("change", (e) => handleBamFileInputChange(e));
                    });

                    bamIndexFileInputsElements.forEach((element) => {
                        element.addEventListener("change", (e) =>
                            handleBamIndexFileInputChange(e)
                        );
                    });

                    loadTrackBtnElements.forEach((element) => {
                        element.addEventListener("click", (e) =>
                            handleLoadTrackBtnClick(e)
                        );
                    });

                    const addViewInIgvButtons = () => {

                        svTableRowElements.forEach((element) => {
                            const tds = element.children;
                            const chrom1 = String(tds[0].innerHTML).trim();
                            const pos1 = parseInt(tds[1].innerHTML);
                            const chrom2 = String(tds[2].innerHTML).trim();
                            const pos2 = parseInt(tds[3].innerHTML);

                            const viewInIgvBtn1 = document.createElement("button");
                            viewInIgvBtn1.innerHTML = "View in IGV";
                            viewInIgvBtn1.className = "btn btn-sm btn-secondary py-0 px-1 my-0 mx-1"
                            viewInIgvBtn1.addEventListener("click", (e) => igvController.setLocusBySpotAndPadding(chrom1, pos1, 2000))

                            const viewInIgvBtn2 = document.createElement("button");
                            viewInIgvBtn2.innerHTML = "View in IGV";
                            viewInIgvBtn2.className = "btn btn-sm btn-secondary py-0 px-1 my-0 mx-1"
                            viewInIgvBtn2.addEventListener("click", (e) => igvController.setLocusBySpotAndPadding(chrom2, pos2, 2000))

                            tds[1].appendChild(viewInIgvBtn1)
                            tds[3].appendChild(viewInIgvBtn2)
                        })

                        cnaTableRowElements.forEach((element) => {
                            const tds = element.children;
                            const chrom = String(tds[0].innerHTML).trim();
                            const pos1 = parseInt(tds[1].innerHTML);
                            const pos2 = parseInt(tds[2].innerHTML);

                            const viewInIgvBtn1 = document.createElement("button");
                            viewInIgvBtn1.innerHTML = "View in IGV";
                            viewInIgvBtn1.className = "btn btn-sm btn-secondary py-0 px-1 my-0 mx-1"
                            viewInIgvBtn1.addEventListener("click", (e) => igvController.setLocusBySpotAndPadding(chrom, pos1, 2000))

                            const viewInIgvBtn2 = document.createElement("button");
                            viewInIgvBtn2.innerHTML = "View in IGV";
                            viewInIgvBtn2.className = "btn btn-sm btn-secondary py-0 px-1 my-0 mx-1"
                            viewInIgvBtn2.addEventListener("click", (e) => igvController.setLocusBySpotAndPadding(chrom, pos2, 2000))

                            tds[1].appendChild(viewInIgvBtn1)
                            tds[2].appendChild(viewInIgvBtn2)
                        })

                        somaticSnvTableRowElements.forEach((element) => {
                            const tds = element.children;
                            const chrom = String(tds[0].innerHTML).trim();
                            const pos = parseInt(tds[1].innerHTML);

                            const viewInIgvBtn = document.createElement("button");
                            viewInIgvBtn.innerHTML = "View in IGV";
                            viewInIgvBtn.className = "btn btn-sm btn-secondary py-0 px-1 my-0 mx-1"
                            viewInIgvBtn.addEventListener("click", (e) => igvController.setLocusBySpotAndPadding(chrom, pos, 2000))

                            tds[1].appendChild(viewInIgvBtn)
                        })
                    }

                    igvController.setOnOpenIGVCompleteCallback(addViewInIgvButtons)
                </script>
            </body>
        </html>
    
